lib/src/codegen.dart

```diff
@@ -51,6 +65,57 @@ String _baseExtendName(String extendName, Map<String, Element> allElements) {
   return baseExtendName;
 }
 
+_PropertyGenerator _propertyGenerator(Map<String,dynamic> classOverrides) {
+  if (classOverrides==null) {
+    return _generateProperty;
+  }
+
+  // Override
+  return (Property property, StringBuffer sb, String getDartName(String)) {
+    var propOverrides= classOverrides[property.name];
+    if (propOverrides == null) {
+      _generateProperty(property,sb,getDartName);
+    } else if (propOverrides is Map) {
+      if (propOverrides.containsKey('get')) {
+        sb.writeln();
+        _writeLines(sb,propOverrides['get']);
+      }
+      if (propOverrides.containsKey('set')) {
+        sb.writeln();
+        _writeLines(sb,propOverrides['set']);
+      }
+    } else {
+      sb.writeln();
+      _writeLines(sb,propOverrides);
+    }
+  };
+}
+
+_writeLines(StringBuffer sb,x) {
+  if (x is  List) {
+    x.forEach((String line) => sb.writeln(line));
```

Comment:
But I wanted to support single line override without having to use a list in that case. I may check if `x` is a `String` and otherwise throw an error though